open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    
</pre><p>You can actually create a function as <code>create_sub_plot_2_grid(df, column_name, plot_x_pos, plot_y_pos, plot_title, plot_kind, plot_color=None, plot_align=None)</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>